### ISSUE TITLE
Exclude unsupported projects when run on a solution

### ISSUE THREAD

**Issue by thomaslevesque** [jerriep/dotnet-outdated#58]
If I run dotnet-outdated on a solution that contains database projects (.sqlproj), I get the following error:

```
Analyzing project and restoring packages...Unable to process the the project `C:\Projects\Foo\Foo.sln. Are you sure this is a valid .NET Core or .NET Standard project type?

Here is the full error message returned from the Microsoft Build Engine:

Microsoft (R) Build Engine version 15.8.166+gd4e8d81a88 for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

C:\Projects\Foo\database\Foo.Customer.Database\Foo.Customer.Database.sqlproj(57,3): error MSB4019: The imported project "C:\Program Files\dotnet\sdk\2.1.400\Microsoft\VisualStudio\v11.0\SSDT\Microsoft.Data.Tools.Schema.SqlTasks.targets" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.
C:\Projects\Foo\database\Foo.Common.Database\Foo.Common.Database.sqlproj(57,3): error MSB4019: The imported project "C:\Program Files\dotnet\sdk\2.1.400\Microsoft\VisualStudio\v11.0\SSDT\Microsoft.Data.Tools.Schema.SqlTasks.targets" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.
```

I understand that dotnet-outdated can't process these projects, but could they be ignored instead of reporting an error?

**Comment by jerriep** [comment]
@thomaslevesque This is now available in V2.1.0 on NuGet

**Comment by thomaslevesque** [comment]
Nice, thanks!
